src/app/shared/components/VueGridRow/VueGridRow.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div :class="$style.vueGridRow">
+    <slot/>
+  </div>
+</template>
+
+<script lang="ts">
+  export default {
+    name:       'VueGridRow',
+  };
+</script>
+
+<style lang="scss" module>
+  @import "../../styles";
+
+  .vueGridRow {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 (-$grid-unit * 2);
```

Comment:
you should use the `$screen-*-gutter` variables for every breakpoint just to make sure that everything fits together

Reply:
ah you'r completely right... done!